core/src/main/kotlin/pages/PageBuilder.kt

```diff
@@ -54,24 +57,52 @@ class DefaultPageBuilder(
         }
     }
 
-    private fun contentForClass(c: Class) = group(c) {
-        header(1) { text(c.name) }
-        c.inherited.takeIf { it.isNotEmpty() }?.let {
-            header(2) { text("SuperInterfaces") }
-            linkTable(it)
+    private fun PageContentBuilder.headerForClasslike(c: ClassLike) =
+        header(1) {
+            text(
+                "${when (c) {
+                    is Class -> "class "
+                    is Enum -> "enum "
+                    else -> ""
+                }}${c.name}"
```

Comment:
String interpolation should only be used with simple cases, not a whole `when` statement

Reply:
Changed.